app/views/content_items/guide.html.erb

```diff
@@ -19,13 +19,17 @@
     <%= render 'govuk_publishing_components/components/title', { title: @content_item.content_title } %>
 
     <% if @content_item.show_guide_navigation? %>
+      <%= render "govuk_publishing_components/components/skip_link", {
+        text: "Skip to contents of guide",
+        href: "#guide-contents"
+      } %>
       <aside class="part-navigation-container" role="complementary">
```

Comment:
I looks like the `aside` overlaps slightly with the focussed skip link:

![image](https://user-images.githubusercontent.com/1732331/92608817-b77e6780-f2ad-11ea-80ad-23ada6acaa39.png)